### ISSUE TITLE
import error when trying new release

### ISSUE THREAD

**Issue by devboell** [danigb/tonal#40]
Hi Dani, how are you?

I thought I'd try the new version, but I get an error when I import tonal.
F.i. if I try to use the `tonal-note` package,

```
import * as tonal from 'tonal'
console.log('tonal.note.midi', tonal.note.midi('C3'))
```
 I get:

```
dev/experiments/tonal-exp/node_modules/tonal-note/index.js:36
    export function tokenize(str) {
    ^^^^^^
    
    SyntaxError: Unexpected token export
      
      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
      at Object.<anonymous> (node_modules/tonal/build/index.js:5:13)
      at Object.<anonymous> (src/tests/index.test.js:2:14)
```

It seems that `tonal/index.js` is exporting the `tonal-note/index.js`, and not the transpiled `tonal-note/build/index.js`. Could that be the reason?

**Comment by danigb** [comment]
Yes, you're right. The transpiled version didn't transpiled the children packages. I'll fix it right now.

**Comment by danigb** [comment]
@devboell I've just published `tonal-1.0.0-1` that should solve this issue. 

But, as far as I know about es6 modules, if you are not transpiling your sources, you must use require: `const tonal = require('tonal')` so it uses the `build/transpiled.js` version.

Hope it helps.

**Comment by devboell** [comment]
Great, it works now.
